packages/kbn-shared-ux-components/src/index.ts

```diff
@@ -58,6 +58,23 @@ export const LazyNoDataViews = React.lazy(() =>
  */
 export const NoDataViews = withSuspense(LazyNoDataViews);
 
+/**
+ * A pure `NoDataViews` component, with no services hooks. Consumers should use `React.Suspennse` or the
```

Comment:
The component already has its own .mdx file, so I am not particularly keen in duplicating it.

Reply:
We can consider removing all of these repetitive blurbs in favor of `@see /path/to/mdx`

Reply:
The reason I was saying is that, as a developer, nothing beats TSDoc for documentation. Mdx are nice but with TSDoc we have a first row seat to help devs 😊 

<img width="1039" alt="Screenshot 2022-03-21 at 09 06 13" src="https://user-images.githubusercontent.com/2854616/159231550-bcb8acad-44b6-47b2-b144-bd10fd7b1e77.png">

I don't think it would hurt to add the _why_ (what's the purpose) of a component from our public API.